handlers.go

```diff
@@ -58,3 +59,53 @@ func Run(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "writing response", http.StatusInternalServerError)
 	}
 }
+
+func Fmt(w http.ResponseWriter, r *http.Request) {
```

Comment:
Looks like a lot of the code is shared here. Maybe we can extract that into a function?

Reply:
https://github.com/gsquire/zig-play/pull/11/commits/d7993dd5316df690512b0266b370c29b68d58d40